src/Integration.Vsix/SonarLint.vsct

```diff
@@ -20,12 +20,48 @@
   <Include href="Commands\PackageCommands.vsct"/>
```

Comment:
Quick note, without adding any command buttons to the menu the menu does not show up so you won't actually find it in "extensions" until you add a command.

Reply:
That must've been fun to debug :)